pdns/pdns_recursor.cc

```diff
@@ -715,10 +713,10 @@ LWResult::Result arecvfrom(std::string& packet, int flags, const ComboAddress& f
 
     *d_len=packet.size();
 
-    if (*nearMissLimit && pident.nearMisses > *nearMissLimit) {
-      g_log<<Logger::Error<<"Too many ("<<pident.nearMisses<<" > "<<*nearMissLimit<<") bogus answers for '"<<domain<<"' from "<<fromaddr.toString()<<", assuming spoof attempt."<<endl;
+    if (nearMissLimit > 0 && pident.nearMisses > nearMissLimit) {
+      g_log<<Logger::Error<<"Too many ("<<pident.nearMisses<<" > "<<nearMissLimit<<") bogus answers for '"<<domain<<"' from "<<fromaddr.toString()<<", assuming spoof attempt."<<endl;
```

Comment:
Is this `bogus` in the sense of dnssec or is this in the sense of "not even close to right"? If the latter, it might be worth changing the wording while we're here. ("invalid", "improper", "bad <field name>", etc.)

Reply:
Yes the word bogus is misleading. It's more "close but no cigar" ;-)

Reply:
I'll update it to "Too many ($attempts > $limit) answers with a wrong message ID for $domain from $source, assuming spoof attempt.".

Reply:
yay thanks!